tools/stress-cluster/cluster/kubernetes/stress-test-addons/values.yaml

```diff
@@ -1,46 +1,46 @@
 env: test
 appInsightsKeySecretName:
-  test: appInsightsInstrumentationKey-tbiruti6oi24k
+  pg: appInsightsInstrumentationKey-s7b6dif73rup6
```

Comment:
If you're going to replace test in this PR, then let's also remove `test.json` in this PR.